### PR TITLE
added tdengine.mdx doc

### PR DESCRIPTION
**Pull request by ankeshmahto**
## Description

Create a doc page for the TDEngine data integration

**Fixes** #5147 

## Type of change

- [ ] 📄 This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have updated the documentation, or created issues to update them.

### CODE REVIEW

**Comment by ankeshmahto** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by ankeshmahto** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/app-integrations/tdengine.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* `password`: password to authenticate your access
+* `url`: Url to TDEngine server. For local server url is localhost:6041 (Default)
+* `token`: Unique token provide while using TDEngine Cloud.
+* `database`: Database name to be connected
+
+
+## Usage
+In order to make use of this handler and connect to TDEngine in MindsDB, the following syntax can be used,
+
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/app-integrations/tdengine.mdx:36]: Please use the below code block:

> \```sql
> SQL GOES HERE
> \```

File: docs/app-integrations/tdengine.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    "database":"test"
+};
+~~~~
+
+
+**Note :-> You can sepecify token instead of user and password while using TDEngine.**
+
+
+Now, you can use this established connection to query your database as follows,
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/app-integrations/tdengine.mdx:53]: Please use the below code block:

> \```sql
> SQL GOES HERE
> \```

**Comment by martyna-mindsdb** [comment]
@ankeshmahto 
Please implement the above review comments in this PR.

**Comment by ankeshmahto** [comment]
@martyna-mindsdb can you please review my pr

**Pull request merged**